### PR TITLE
Add scripts to print startup time and memory usage

### PR DESCRIPTION
**Pull request by mraible**
For this to work on Linux, `gdate` will likely need to be replaced with `date` and `fkill` will need to replaced with another command.

To test this, run `build.sh` to build all the projects, then try one of the following commands:

```
./start.sh micronaut
./start.sh micronaut-o
./start.sh quarkus
./start.sh spring-boot
./start.sh helidon
```

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    echo "Selected app is not found in the options map."
+    exit 1
+    ;;
+esac
+
+date_cmd="date"
+kill_cmd="fuser -k -n tcp 8080"
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+   echo "gdate is used to record timestamps on macOS. Install it using 'brew install coreutils' if needed."
```
Comment (mraible) [inline start.sh:40]: @deepu105 Do you know how to conditionally print this message? I'd rather it's hidden if `gdate` is also installed.

File: memory.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+# Define memory function
+memory() {
+  ps -o pid,rss,command | grep $1 | awk '{$2=int($2/1024)"M";}{ print;}'
+}
+
+kill_cmd="fuser -k -n tcp 8080"
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+   kill_cmd="fkill :8080"
```
Comment (mraible) [inline memory.sh:48]: `fkill` is just a Node module I use. You can install it with `npm i -g fkill-cli`. Will `fuser` work on macOS? If not, we should prompt the user to install `fkill-cli` if the `fkill` command doesn't exist.

Comment (deepu105) [reply memory.sh:48]: Lemme check. I didn't test this command on mac

File: start.sh (hunk 1 above)
Comment (deepu105) [reply start.sh:40]: I'll add it

File: memory.sh (hunk 2 above)
Comment (deepu105) [reply memory.sh:48]: So fuser doent work on mac but `kill -9 $(lsof -i:PORT -t) 2> /dev/null` works natively

**Pull request merged**